tmt/steps/__init__.py

```diff
@@ -101,6 +103,12 @@
             or previous command line options. Use --name to specify which one, or omit --name and
             update all existing phases.
             """),
+    option(
+        '--allowed-how',
+        metavar='PATTERN',
+        default=None,
+        help='If set, only ``how`` matching pattern is allowed.'
```

Comment:
```suggestion
        help='If set, only ``how`` matching give regular expression is allowed.'
```

What about being a bit more specific about the pattern?

Reply:
Changed in 97885b1